.github/workflows/checker.yml

```diff
@@ -0,0 +1,114 @@
+name: Checker Script CI
+
+on:
+  pull_request:
+    branches:
+      - main
```

Comment:
I don't think we need main here. Only the `master` branch exists.

Reply:
Makes sense!  Thanks ser!

Reply:
Done ser!